Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -245,19 +245,6 @@ void ABenchmarkGymGameMode::SpawnNPC(const FVector& SpawnLocation)
 
 void ABenchmarkGymGameMode::StartPlay()
 {
-	UE_LOG(LogBenchmarkGym, Log, TEXT("Enabling custom density spawning."));
```

Comment:
How are NPCs spawned now?

Reply:
When CheckInitCustomSpawning is called by FindPlayerStart_Implementation, the problem with having it here is that the NetDriver/WorkerFlags class is not initialized yet!

Reply:
It was also double-spawning NPCs here because CheckInitCustomSpawning will also do that (it has a bool latch to ensure it only runs once)

Reply:
Okay - perhaps just remove this `StartPlay` funciton entirely then?